x-pack/test/security_solution_cypress/cypress/e2e/detection_response/detection_engine/exceptions/alerts_table_flow/endpoint_exceptions.cy.ts

```diff
@@ -99,7 +101,8 @@ describe.skip(
       validateExceptionConditionField('file.Ext.code_signature');
       addExceptionFlyoutItemName(ITEM_NAME);
 
-      cy.get(ADD_AND_BTN).click();
+      // Add non-nested condition
+      cy.get(ADD_NESTED_BTN).click();
```

Comment:
This seems contradictory. 🤔 Nested button adds a nested condition isn't it?

Reply:
yep, seems contradictory, but behind the scenes there seems to be one button with the text 'Add nested entry'/'Add non-nested entry' depending on the last entry